cairo/src/font/glyph.rs

```diff
@@ -8,7 +8,7 @@ use std::fmt;
 pub struct Glyph(ffi::cairo_glyph_t);
 
 impl Glyph {
-    pub fn index(&self) -> u64 {
+    pub fn index(&self) -> libc::c_ulong {
```

Comment:
`unsigned long` :scream:

Reply:
seems like it https://www.cairographics.org/manual/cairo-text.html#cairo-glyph-t, no idea why

Reply:
Historical reasons